src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+        private Person person;
+        private Tag tag;
+        private String tagOperation;  
```

Comment:
string is too generic for this; think how else you can represent this information.